docs/releasing.md

```diff
@@ -50,9 +50,9 @@ yarn build-all
 
 # Verify the viewer will work.
 yarn serve-viewer
-# Works with v4 report? http://localhost:8000/viewer?gist=7251f9eba409f385e4c0424515fe8009
```

Comment:
v4 report wasn't working. I don't think we need to support this anymore.

Reply:
Ideally we can avoid contributing to URL rot. https://googlechrome.github.io/lighthouse/viewer/?gist=7251f9eba409f385e4c0424515fe8009 is working (currently at 9.1.0), so it was a change in this release?

Reply:
actually, https://lighthouse-e6g7mely5-googlechrome.vercel.app/gh-pages/viewer/?gist=7251f9eba409f385e4c0424515fe8009 also seems to work from this deploy. Some local issue?

Reply:
what's the error? seems good to me

Reply:
Ope false alarm. I thought it didn't generate the opportunities or diagnostics section. Turns out it just passed everything and I missed the passed audit section 🙃 .